tool/manual_gen/win32api.json

```diff
@@ -1335,6 +1335,11 @@
         "dllLibrary": "gdi32",
         "comment": "The ExtCreatePen function creates a logical cosmetic or geometric pen that has the specified style, width, and brush attributes."
     },
+    "ExtractAssociatedIconW": {
```

Comment:
Rename ExtractAssociatedIconW to ExtractAssociatedIcon

Reply:
Done